src/libsystemd-network/dhcp6-lease-internal.h

```diff
@@ -14,6 +14,14 @@
 #include "macro.h"
 #include "time-util.h"
 
+typedef struct DHCP6VendorOption {
```

Comment:
We already have sd_dhcp6_option type for vendor option. Please reuse it.
See sd_dhcp6_client_add_vendor_option(), dhcp6_option_append_vendor_option(), and so on.

Reply:
Fixed it